mitmproxy/flow.py

```diff
@@ -126,7 +126,7 @@ def __init__(
         self._backup: Flow | None = None
         self.marked: str = ""
         self.is_replay: str | None = None
-        self.metadata: Any = None
```

Comment:
if i change this to a lot of test start failing so should a leave metadata as empty `dict` or it must initialized with `None`
```
self.metadata: Any = None
```